diff --git a/kqueue.c b/kqueue.c
index 35b3f2b..cf425cf 100644
--- a/kqueue.c
+++ b/kqueue.c
@@ -28,6 +28,7 @@
 
 #include <sys/types.h>
 #include <sys/event.h>
+#include <sys/time.h>
 #include <assert.h>
 #include <errno.h>
 #include <stdlib.h>
diff --git a/metadata.c b/metadata.c
index 04ed27c..63d50dc 100644
--- a/metadata.c
+++ b/metadata.c
@@ -1488,9 +1488,9 @@ GetVideoMetadata(const char *path, const char *name)
 			while( (tag = av_dict_get(ctx->metadata, "", tag, AV_DICT_IGNORE_SUFFIX)) )
 			{
 				//DEBUG DPRINTF(E_DEBUG, L_METADATA, "  %-16s: %s\n", tag->key, tag->value);
-				if( strcmp(tag->key, "title") == 0 )
-					m.title = escape_tag(trim(tag->value), 1);
-				else if( strcmp(tag->key, "genre") == 0 )
+				if( strcmp(tag->key, "title") == 0 ) {
+					// m.title = escape_tag(trim(tag->value), 1);
+                } else if( strcmp(tag->key, "genre") == 0 )
 					m.genre = escape_tag(trim(tag->value), 1);
 				else if( strcmp(tag->key, "artist") == 0 )
 					m.artist = escape_tag(trim(tag->value), 1);
